tools/deviceGen/output/features_intersect.h

```diff
@@ -0,0 +1,78 @@
+   // VK 1.0 Core
+    res.robustBufferAccess &= _rhs.robustBufferAccess;
+    res.geometryShader &= _rhs.geometryShader;
+    res.tessellationShader &= _rhs.tessellationShader;
+    res.depthBounds &= _rhs.depthBounds;
+    res.wideLines &= _rhs.wideLines;
+    res.largePoints &= _rhs.largePoints;
+    res.alphaToOne &= _rhs.alphaToOne;
+    res.textureCompressionETC2 &= _rhs.textureCompressionETC2;
+    res.textureCompressionASTC_LDR &= _rhs.textureCompressionASTC_LDR;
+    res.textureCompressionBC &= _rhs.textureCompressionBC;
+    res.pipelineStatisticsQuery &= _rhs.pipelineStatisticsQuery;
+    res.vertexPipelineStoresAndAtomics &= _rhs.vertexPipelineStoresAndAtomics;
+    res.fragmentStoresAndAtomics &= _rhs.fragmentStoresAndAtomics;
+    res.shaderTessellationAndGeometryPointSize &= _rhs.shaderTessellationAndGeometryPointSize;
+    res.shaderStorageImageMultisample &= _rhs.shaderStorageImageMultisample;
+    res.shaderStorageImageReadWithoutFormat &= _rhs.shaderStorageImageReadWithoutFormat;
+    res.shaderStorageImageArrayDynamicIndexing &= _rhs.shaderStorageImageArrayDynamicIndexing;
```

Comment:
looking at other comments, these were either no supposed to be exposed or limits